x-pack/solutions/observability/plugins/slo/public/types.ts

```diff
@@ -106,6 +107,7 @@ export interface SLOPublicPluginsStart {
   security?: SecurityPluginStart;
   fieldsMetadata: FieldsMetadataPublicStart;
   apmSourcesAccess: ApmSourceAccessPluginStart;
+  contentManagement: ContentManagementPublicStart;
```

Comment:
Linking with this [issue](https://github.com/elastic/kibana/issues/229936#issue-3276577100), where we can remove the dependency to the contentManagement plugin

Reply:
Just to clarify, this comment of mine regarding removing dependency to the contentManagement plugin, is something that does NOT need to be addressed in current PR. I put it here as a reminder for when addressing [this](https://github.com/elastic/kibana/issues/229936#issue-3276577100) issue.

I also just read Dominique's [here](https://github.com/elastic/kibana/issues/229936#issuecomment-3175359317), so it might be that we don't need to replace anything (I need to refresh what it is about). @cesco-f Can you weigh in here if we can replace the use of dashboardServiceProvider with the api in the rules form?